Casks/p/pronterface.rb

```diff
@@ -1,13 +1,13 @@
 cask "pronterface" do
-  version "2.0.1,3.10"
-
   on_big_sur :or_older do
+    version "2.0,3.10"
     sha256 "5c66655efd1f5e8906169467d9843e55ec5a7f9b72fc0d827e59b850ebf944fb"
 
     url "https://github.com/kliment/Printrun/releases/download/printrun-#{version.csv.first}/printrun-#{version.csv.first}_macos-11_x64_py#{version.csv.second}.zip"
   end
```

Comment:
```suggestion

    livecheck do
      skip "Legacy version"
    end
  end
```
We need to skip the `livecheck` on the legacy version.

Reply:
Thanks, I reverted this change. I thought they had dropped support in 2.0.1, but that was for macOS 10. The release for macOS 11 seems to (still) be missing.

Reply:
Can you file an issue upstream to find out what's happening?

Reply:
The release for macOS 11 was dropped since there should be no difference between the two versions (see [comment](https://github.com/kliment/Printrun/issues/1427#issuecomment-2137875464)).